website/content/docs/get-started/developer-qs.mdx

```diff
@@ -72,153 +46,7 @@ Now, let's add the import statements for the client library to the top of the fi
 <CodeBlockConfig heading="import statements for client library" lineNumbers>
 
 ```go
-import vault "github.com/openbao/openbao/api"
```

Comment:
Curious, why doesn't this work? It might not work right now, but it should definitely work in general.

Reply:
Ah I see, you left it at the end, its no longer `vault` :-D 

But in general, I think we do want to mention that we're still planning on being compatible with Vault client libraries.